ThemeDecorator/ThemeDecorator.module.less

```diff
@@ -52,8 +54,6 @@
 	:focus {
 		// Prevent Chrome's default focus treatment
 		outline: none;
-		// Prevent Android Chrome from highlighting tapped elements
-		-webkit-tap-highlight-color: transparent;
```

Comment:
why this change is different with https://github.com/enactjs/sandstone/pull/784??

Reply:
It was missing in #784.